cmd/exec.go

```diff
@@ -20,6 +20,7 @@ import (
 	"os"
 	"time"
 
+	"github.com/kballard/go-shellquote"
```

Comment:
I think this file is not being used anymore. Now everything goes through `exec` package, right?